GermanFilter/sections/specific.txt

```diff
@@ -61,6 +61,7 @@ ecowoman.de##.ad-info
 ecowoman.de###article-products.nl-gp-a-l
 englisch-hilfen.de##.top-banner
 ln-online.de##.pdb-div-gpt-footerad
+online-umwandeln.de##.lead-responsive
```

Comment:
online-umwandeln.de###content-area > .entry > small[style="font-size:10px"]
online-umwandeln.de###content-area > .entry > small[style="font-size:10px"] + .lead-responsive

First rule is hiding "Anzeige" which is translated "Ad".

Reply:
thanks